README.md

```diff
@@ -1,25 +1,211 @@
 
-The contents of this Github repository are freely available for reuse under the [MIT license](https://opensource.org/licenses/MIT). Data in the CZ Cell Census are available for re-use under the [CC-BY license](https://creativecommons.org/licenses/by/4.0/).
+- Cloud-based data storage and access.
+- Efficient access for larger-than-memory slices of data.
+- Data streaming for iterative/parallelizablne  methods.
+- R and Python support.
+- Export to AnnData and Seurat.
 
-### For More Help
 
-Please file a issue on this repo, or contact us at <cellxgene@chanzuckerberg.com>
+The Cell Census is free to use.
 
-### Security
+## Cell Census data releases
 
-If you believe you have found a security issue, we would appreciate notification. Please send email to <security@chanzuckerberg.com>.
+Starting in  mid 2023, Cell Census long-term supported data builds will be released every 6 months and are guaranteed to be stored for public access for at least 5 years upon release. 
 
-### Code of Conduct
+In between long-term supported data build releases, weekly builds are released without any guarantee of permanence. 
+
+## Cell Census data organization
+
+The Cell Census follows a specific [data schema](https://github.com/chanzuckerberg/cell-census/blob/main/docs/cell_census_schema_0.1.0.md). Briefly, the Cell Census is a collection of a variety of **SOMA** objects organized with the following hierarchy.
+
+
+Cell Census, a collection with:
+
+- `"census_info" ` — collection with summary objects:
+   - `"summary"` — data frame with Cell Census metadata.
+   - `"datasets"` — data frame listing all datasets included.
+   - `"summary_cell_counts"`  — data frame with cell counts across cell metadata variables.
+- `"census_data"` — collection with the single-cell data per organism:
+	- `"homo_sapiens"` or `"mus_musculus"` — collection with:
+		- `obs`  — data frame with cell metadata.
+		- `ms["RNA"]` — collection with the count matrix and gene metadata:
+		   - `X["raw"]` — sparse matrix with raw counts.
+		   - `var` — data frame with gene metadata for >60K genes.
+		   - `"feature_dataset_presence_matrix"`— sparse boolean matrix flagging genes measured per dataset. 
+
+## Getting started
+
+### Requirements
+
+The Cell Census requires a Linux or MacOS system with:
+
+- Python 3.7 to Python 3.10. Or R, supported versions TBD.
+- Recommended: >16 GB of memory.
+- Recommended: >5 Mbps internet connection. 
+
+### Documentation
+
+Reference documentation, data description, and tutorials can be access at the Cell Census doc-site. *Coming soon*. 
+
+Demonstration notebooks can be found [here](https://github.com/chanzuckerberg/cell-census/tree/main/api/python/notebooks).
```

Comment:
Should this be included under the python section since it references only python notebooks?